<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="LoginPage">
    <widget class="QLabel" name="label_noUserLogin_loginPage_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Ubuntu&quot;;
background-color: rgb(240, 240, 240);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_dateTime_loginPage_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>40</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(240, 240, 240);
font: 12pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_userLogin_loginPage_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>0</y>
       <width>581</width>
       <height>36</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ubuntu</family>
       <pointsize>16</pointsize>
       <weight>7</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 230, 230);
font: 57 16pt &quot;Ubuntu&quot;;</string>
     </property>
     <property name="text">
      <string>User Login</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_bioMetricLogin_loginPage_2">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>260</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>BIOMETRIC LOGIN</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_loginPassword_loginPage_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>210</y>
       <width>111</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #000000;
background-color: rgb(240, 240, 240);
border-color: rgb(240, 240, 240);
</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Desktop/NEW EKF24/EKF04/EKF04/ekf04.qrc">
       <normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/Password_active2.bmp</normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/Password_active2.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_loginUserName_loginPage_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>130</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 220);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_loginPassword_loginPage_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>230</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(220, 220, 220);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_biometricLogin_loginPage_2">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>160</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #000000;
background-color: rgb(40, 230, 230);
border-color: rgb(240, 240, 240);
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../Desktop/NEW EKF24/EKF04/EKF04/ekf04.qrc">
       <normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/only hand.bmp</normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/only hand.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_loginUserName_loginPage_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>111</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #000000;
background-color: rgb(240, 240, 240);
border-color: rgb(240, 240, 240);
</string>
     </property>
     <property name="text">
      <string>User Name</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Desktop/NEW EKF24/EKF04/EKF04/ekf04.qrc">
       <normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/User_active.bmp</normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/User_active.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_back_loginPage_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>101</width>
       <height>79</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border:1px solid #000000;
background-color: rgb(40, 230, 230);
border-color: rgb(40, 230, 230);</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Desktop/NEW EKF24/EKF04/EKF04/ekf04.qrc">
       <normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/DASHBOARDTab_back_A.bmp</normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/DASHBOARDTab_back_A.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
    <widget class="QToolButton" name="toolButton_loginMain_loginPage_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>320</y>
       <width>221</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(40, 230, 230);</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Desktop/NEW EKF24/EKF04/EKF04/ekf04.qrc">
       <normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/AutoLogINTime_Active.bmp</normaloff>:/ekf04_Teal_Icons/icons/Teal_backgrund/AutoLogINTime_Active.bmp</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextUnderIcon</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources>
  <include location="../../../Desktop/NEW EKF24/EKF04/EKF04/ekf04.qrc"/>
 </resources>
 <connections/>
</ui>
